server/webapp/WEB-INF/rails.new/app/controllers/api_v1/base_controller.rb

```diff
@@ -0,0 +1,53 @@
+##########################GO-LICENSE-START################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################GO-LICENSE-END##################################
+
+module ApiV1
+  class BaseController < ::ApplicationController
+
+    FORMATS                = [:json_hal_v1]
+    DEFAULT_FORMAT         = FORMATS.last
+    DEFAULT_ACCEPTS_HEADER = Mime[DEFAULT_FORMAT].to_s
+
+    skip_before_filter :verify_authenticity_token
+    before_filter :set_default_response_format
+
+    def set_default_response_format
+      request.format = DEFAULT_FORMAT unless params[:format]
+    end
+
+    def redirect_json_hal(url, status=302)
+      json = {
+        _links: {
+          redirect: {
+            href: url
+          }
+        }
+      }
+
+      response.status = status
+      render json_hal_v1: json, location: url
+    end
+
+    rescue_from RecordNotFound do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.